data/remnant/remnant 2 cognizance.txt

```diff
@@ -1149,7 +1149,7 @@ mission "Remnant: Cognizance 24"
 
 mission "Remnant: Cognizance 25"
 	name "Patrol systems near Postverta."
-	description "Gavriil has asked you to do a patrol through the nearby systems."
+	description "Gavriil has asked you to do a patrol through the nearby systems. Along the way, make sure that a Palavret that passed by recently did not leave behind any surprises."
```

Comment:
```suggestion
	description "Gavriil has asked you to do a patrol through the nearby systems to clear out any Korath presence."
```

Reply:
I see what you mean, I think it can be changed differently though.